docs/source/get_started/installation.rst

```diff
@@ -117,8 +116,8 @@ instructions for adding the package to your system.
 .. _Setuptools Install: https://pypi.python.org/pypi/setuptools/1.1.6
 .. _Pip Install: https://pip.pypa.io/en/stable/installing/
 
-`Ply`_
-~~~~~~
+`Ply`_ and `Cppy`_
```

Comment:
Given how the rest of the document is organized, I am not sure why you chose to group those 2 together. I think it would be better for Cppy to have its own separate entries.